products/cloudiot/terraform.yaml

```diff
@@ -145,7 +145,41 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       credentials.publicKeyCertificate: !ruby/object:Overrides::Terraform::PropertyOverride
         # See the comment on stateNotificationConfig.exclude
         exclude: true
-
+  Device: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: [ "{{%registry}}/devices/{{name}}" ]
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        required: true
+        name: 'name'
+        description: |
+          A unique name for the resource.
+      name: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Can we just remove the `id` field from `api.yaml` altogether and replace it with `name`? It is confusing to declare 2 fields, then exclude one and rename the other to replace the excluded one.

Reply:
I think we should keep both, as the API describes them. Additionally, that's the approach we took for the Device Registry resource (in the same `api.yaml`, up above).

Reply:
Sounds good